internal/mocks/atlas/global_clusters.go

```diff
@@ -0,0 +1,71 @@
+package atlas
+
+import (
+	"context"
+	"fmt"
+
+	"go.mongodb.org/atlas/mongodbatlas"
+)
+
+type GlobalClustersClientMock struct {
+	GetFunc     func(projectID string, clusterName string) (*mongodbatlas.GlobalCluster, *mongodbatlas.Response, error)
+	GetRequests map[string]struct{}
+
+	AddManagedNamespaceFunc     func(projectID string, clusterName string, namespace *mongodbatlas.ManagedNamespace) (*mongodbatlas.GlobalCluster, *mongodbatlas.Response, error)
+	AddManagedNamespaceRequests map[string]*mongodbatlas.ManagedNamespace
+
+	DeleteManagedNamespaceFunc     func(projectID string, clusterName string, namespace *mongodbatlas.ManagedNamespace) (*mongodbatlas.GlobalCluster, *mongodbatlas.Response, error)
+	DeleteManagedNamespaceRequests map[string]*mongodbatlas.ManagedNamespace
+
+	AddCustomZoneMappingsFunc     func(projectID string, clusterName string, request *mongodbatlas.CustomZoneMappingsRequest) (*mongodbatlas.GlobalCluster, *mongodbatlas.Response, error)
+	AddCustomZoneMappingsRequests map[string]*mongodbatlas.CustomZoneMappingsRequest
+
+	DeleteCustomZoneMappingsFunc     func(projectID string, clusterName string) (*mongodbatlas.GlobalCluster, *mongodbatlas.Response, error)
+	DeleteCustomZoneMappingsRequests map[string]struct{}
+}
+
+func (c *GlobalClustersClientMock) Get(_ context.Context, projectID string, clusterName string) (*mongodbatlas.GlobalCluster, *mongodbatlas.Response, error) {
+	if c.GetRequests == nil {
+		c.GetRequests = map[string]struct{}{}
+	}
+
+	c.GetRequests[fmt.Sprintf("%s.%s", projectID, clusterName)] = struct{}{}
+
+	return c.GetFunc(projectID, clusterName)
+}
+func (c *GlobalClustersClientMock) AddManagedNamespace(_ context.Context, projectID string, clusterName string, namespace *mongodbatlas.ManagedNamespace) (*mongodbatlas.GlobalCluster, *mongodbatlas.Response, error) {
```

Comment:
gotfmt did not complain?

```suggestion
}

func (c *GlobalClustersClientMock) AddManagedNamespace(_ context.Context, projectID string, clusterName string, namespace *mongodbatlas.ManagedNamespace) (*mongodbatlas.GlobalCluster, *mongodbatlas.Response, error) {
```